src/Token/TokenBase.tsx

```diff
@@ -0,0 +1,93 @@
+import {KeyboardEvent} from 'react'
+import styled from 'styled-components'
+import {variant} from 'styled-system'
+import {get} from '../constants'
+
+export type TokenSizeKeys = 'sm' | 'md' | 'lg' | 'xl'
```

Comment:
Should we use full names? small, medium, large, xlarge?

Looking around at other components, that seems to be the pattern: [CircleBadge](https://github.com/primer/react/blob/627cd3e95614ab4347e5acf456dd23588196ec87/src/CircleBadge.tsx#L8-L11), [Label](https://github.com/primer/react/blob/627cd3e95614ab4347e5acf456dd23588196ec87/src/Label.tsx#L18), [Spinner](https://github.com/primer/react/blob/627cd3e95614ab4347e5acf456dd23588196ec87/src/Spinner.tsx#L7-L11)

Part 2: How do you feel about using `variant` instead of `size` here, like we do for [Label](https://primer.style/react/Label#default-example).

Reply:
If other components use "small", "medium", "large" - let's do that.

@colebemis specifically requested we use `size` instead of `variant` (which is what I originally had). I think `size` makes more sense, and we should keep it.

Reply:
+1 to using the full names for sizes (`small`, `medium`, `large`)

IMO `Label` should use `size` instead of `variant` as well.